### DISCUSSION TITLE
Adding CEXs Bitcoin Wallets

### DISCUSSION THREAD

**Discussion by Tomwhh** [duneanalytics/spellbook#3048] [Sector: Labels]
Dear Sers,

I will enhance the Exchanges PoR Dashboard (https://dune.com/21shares_research/cexs-proof-of-reserve) in the future and would love to contribute to the labels as well so anyone can query it easily. Thus, I wish to add a new cex label view for Bitcoin, and add Bybit to the exiting label view for Etheruem

Cc: @hildobby @andrewhong5297 @soispoke 

tyvm

**Comment by andrewhong5297** [comment]
Let's do it! Have you contributed to spellbook before? If not, I can get you started with the schema/file setup so then you only need to add the addresses/cex names

**Reply by Tomwhh** [reply]
I only contributed to the price table previously. So I would be more than grateful if you can help me with the scheme and file setup ser!

**Reply by andrewhong5297** [reply]
got it - can you give me a few addresses with CEX names? I will kick start it with a few rows and then you can add more :)

**Reply by andrewhong5297** [reply]
3 or 4 is fine

**Reply by Tomwhh** [reply]
Thank you ser! Below are Binace's Bitcoin addresses

'34xp4vRoCGJym3xR7yCVPFHoCNxv4Twseo','38ztxG7GL1LEEbC9gKpqEKEh7WZ3KDTLMi',
'399QCnqVzAt4HGU1SV7PfVPYovb1BX3u9Y','3HdGoUTbcztBnS7UzY4vSPYhwr424CiWAA'

**Reply by Tomwhh** [reply]
Here is my source:
https://portfolio.nansen.ai/dashboard/binance

**Reply by andrewhong5297** [reply]
okay got it! will have something for you to contribute to soon, I've added them in here but the we're still fixing some things in the other parts of [the PR ](https://github.com/duneanalytics/spellbook/pull/3003)

**Reply by Tomwhh** [reply]
thanks a lot Andrew appreciate it!

**Reply by andrewhong5297** [reply]
hey! I've added the ones you mentioned here, if you want just to contribute directly there now. lmk if you run into issues https://github.com/duneanalytics/spellbook/blob/main/models/labels/addresses/institution/identifier/cex/labels_cex_bitcoin.sql
